Composer/packages/server/src/models/connector/csharpBotConnector.ts

```diff
@@ -117,73 +124,27 @@ export class CSharpBotConnector implements IBotConnector {
     return new Promise((resolve, reject) => {
       const runtime = spawn(
         'dotnet',
-        [
-          'bin/Debug/netcoreapp2.1/BotProject.dll',
-          `--urls`,
-          this.endpoint,
-          ...this.getConnectorConfig(config),
-          '--environment',
-          'development',
-        ],
+        ['bin/Debug/netcoreapp2.1/BotProject.dll', `--urls`, this.endpoint, ...this.getConnectorConfig(config)],
```

Comment:
Should this be more defensive against `this.endpoint` not being set correctly until connect has been called?

Reply:
`this.endpoint` is the value from environment config. And default environment config is set in settings/index.ts. Seem like it's a hard code value. Do I need to validate the endpoint in csharpBotConnector? 
![image](https://user-images.githubusercontent.com/21174180/70418815-1c96f980-1a9f-11ea-9525-e8ca621238c4.png)

Reply:
Maybe we should check whether the endpoint is valid earlier, suck like in env.ts?